.editorconfig

```diff
@@ -94,3 +94,6 @@ trim_trailing_whitespace = unset
 
 [pkgs/top-level/emscripten-packages.nix]
 trim_trailing_whitespace = unset
+
+[pkgs/development/tools/gofumpt/default.nix]
+trim_trailing_whitespace = unset
```

Comment:
Please try this instead:

https://github.com/editorconfig-checker/editorconfig-checker#excluding-blocks

Reply:
Exactly what I was originally looking for! Thanks!